src/lib/backend_fapi.c

```diff
@@ -68,6 +158,14 @@ static char * tss_path_from_id(unsigned id, const char *type) {
     return path;
 }
 
+static char * path_get_parent(const char *path) {
```

Comment:
pointer * tight to the variable... `static char *path_get_parent()` Here and throughout.

Reply:
I think I don't understand ?

Reply:
Sorry mispoke, * tight to the fn name... I edited my comment to make seem more awake.